scripts/dts/extract/edts.py

```diff
@@ -23,6 +23,9 @@ def edts_insert_device_property(node_address, property_path, property_value):
     device_id = edts_device_id(node_address)
     edts.insert_device_property(device_id, property_path, property_value)
 
+def edts_insert_device_type(compatible, device_type):
```

Comment:
Remove - use edts_insert_device_property with 'device-type/{}'.format(device_type) instead.

Reply:
I'm already using edts_insert_device_property with 'device-type/{}'.format(device_type)
in heuristics class.
Aim of edts_insert_device_type is not to fill in device-type sub-node property but dedicated device-type dict. This new dict should be used at least for CI and potentially other purposes related to https://github.com/zephyrproject-rtos/zephyr/issues/9406.